apps/src/lib/node/ledger/shell/queries.rs

```diff
@@ -47,7 +48,7 @@ pub enum Error {
     )]
     NotValidatorKeyHash(String, Epoch),
     #[error("Invalid validator tendermint address")]
-    InvalidTMAddress,
+    InvalidTMAddress(#[from] Utf8Error),
```

Comment:
This shouldn't land as is, just added this to figure out what was going on. I think we will need to fix https://github.com/anoma/namada/issues/200 then remove this error